build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -266,5 +269,25 @@ private static void FixPermissions(string directory)
                 FS.FixModeFlags(directory);
             }
         }
+
+        private static string GetSharedFrameworkPublishPath(BuildTargetContext c)
+        {
+            string sharedFrameworkPublishPath = string.Empty;
+            
+            string preBuiltPortableStagingPath=c.BuildContext.Get<string>("PortableBuildStagingLocation");
+
+            // set to default path if PORTABLE_BUILD_STAGING_LOCATION environment variable doesn't exist
```

Comment:
I think you can update the comment to be more reflective of the intent - "If we are not generating distro specific installers for portable build, then we wont have access to staging location and thus, will use default   binary location where SharedFX was published".